src/st/entity_axe_knight.h

```diff
@@ -1,37 +1,67 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 
+// Code in this file is very similar to e_blue_axe_knight.h
+// minus some additional attacks and code not present in the higher
+// level blue axe knight
+
 typedef enum {
     AXE_KNIGHT_INIT,
     AXE_KNIGHT_IDLE,
     AXE_KNIGHT_WALK_TOWARDS_PLAYER,
     AXE_KNIGHT_WALK_AWAY_FROM_PLAYER,
     AXE_KNIGHT_STANDING_THROW,
     AXE_KNIGHT_DUCKING_THROW,
-    AXE_KNIGHT_UNUSED, // Charge Attack missing step from the blue AxeKnight
-    AXE_KNIGHT_ARCING_THROW, // Unused, present in the blue AxeKnight
+    AXE_KNIGHT_CHARGE_ATTACK, // Unused, present in the blue AxeKnight
+    AXE_KNIGHT_ARCING_THROW,  // Unused, present in the blue AxeKnight
     AXE_KNIGHT_DYING,
 } EntityAxeKnightSteps;
 
 static s16 sensors_move[] = {0, 32, 8, 0};
 static s16 sensors_ground[4][2] = {{0, 32}, {0, 4}, {8, -4}, {-16, 0}};
 static s16 dead_particle_pos[][2] = {
     {-8, -4}, {8, -6}, {-2, 4}, {-1, -7}, {2, 8}, {11, -7}, {4, 6}, {-3, 0}};
-static u16 unused[] = {
-    0x0114, 0x020A, 0x030A, 0x0414, 0x030A, 0x020A, 0x0000, 0x0000, 0x0508,
-    0x0608, 0x0708, 0x0D08, 0x0E08, 0x0F02, 0x1402, 0x1502, 0x1620, 0x1708,
-    0x0000, 0x0000, 0x0508, 0x0608, 0x0708, 0x0808, 0x090A, 0x0A04, 0x0B04,
-    0x0C04, 0x0000, 0x0000, 0x0E04, 0x0F04, 0x1004, 0x1104, 0x0000, 0x0000,
-    0x0D08, 0x0E08, 0x0F01, 0x1001, 0x1101, 0x1220, 0x1308, 0x0000};
+
+// *********************
+// Unused animations - these are used for the blue axe knight
+// but are present in data here
+// *********************
+static u8 anim_walk_alt[] = {0x14, 0x01, 0x0A, 0x02, 0x0A, 0x03, 0x14, 0x04,
+                             0x0A, 0x03, 0x0A, 0x02, 0x00, 0x00, 0x00, 0x00};
+static u8 anim_throw_stand_alt[] = {
+    0x08, 0x05, 0x08, 0x06, 0x08, 0x07, 0x08, 0x0D, 0x08, 0x0E, 0x02, 0x0F,
+    0x02, 0x14, 0x02, 0x15, 0x20, 0x16, 0x08, 0x17, 0x00, 0x00, 0x00, 0x00,
+};
+static u8 anim_throw_duck_alt[] = {
+    0x08, 0x05, 0x08, 0x06, 0x08, 0x07, 0x08, 0x08, 0x0A, 0x09,
+    0x04, 0x0A, 0x04, 0x0B, 0x04, 0x0C, 0x00, 0x00, 0x00, 0x00};
+static u8 anim_charge_swing_alt[] = {
+    0x04, 0x0E, 0x04, 0x0F, 0x04, 0x10, 0x04, 0x11, 0x00, 0x00, 0x00, 0x00};
+static u8 anim_charge_final_slice_alt[] = {
+    0x08, 0x0D, 0x08, 0x0E, 0x01, 0x0F, 0x01, 0x10,
+    0x01, 0x11, 0x20, 0x12, 0x08, 0x13, 0x00, 0x00};
+// *********************
+// End unused animations
+// *********************
+
 static u8 anim_walk[] = {0x1A, 0x18, 0x0A, 0x19, 0x0A, 0x1A, 0x1A,
                          0x1B, 0x0A, 0x1A, 0x0A, 0x19, 0x00};
 static u8 anim_throw_duck[] = {
-    0x08, 0x1C, 0x08, 0x1D, 0x08, 0x1E, 0x08, 0x1F, 0x0A, 0x20, 0x04, 0x21,
-    0x04, 0x22, 0x04, 0x23, 0x00, 0x00, 0x00, 0x00, 0x08, 0x24, 0x08, 0x25,
-    0x02, 0x26, 0x02, 0x27, 0x02, 0x28, 0x20, 0x29, 0x08, 0x2A, 0x00};
+    0x08, 0x1C, 0x08, 0x1D, 0x08, 0x1E, 0x08, 0x1F, 0x0A, 0x20,
+    0x04, 0x21, 0x04, 0x22, 0x04, 0x23, 0x00, 0x00, 0x00, 0x00};
```

Comment:
Cleaned up these data imports a little - the green axe knight data here was combining two animations, only the first of which was in use. You can see this from the empty byte termination in between the two `0x00, 0x00, 0x00, 0x00`